<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Kruunu2Euro">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Crone2Euro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tulosLaatikko">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Euro2Kruunu">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Euro2Crone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MuunnosTulos">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Muunnettava</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Muunnoksen tulos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
